Fix total size count for data scroller

Samuel-StO opened the pull request:
Fix #11019
Fix #6087 

Use the count method for data scroller when we are in lazy

Samuel-StO committed:
Fix total size count for data scroller

Use the count method for data scroller when we are in lazy Method
Link with : https://github.com/primefaces/primefaces/issues/11019

tandraschko commented on primefaces/src/main/java/org/primefaces/component/datascroller/DataScrollerRenderer.java:
```diff
@@ -192,6 +192,16 @@ protected void encodeScript(FacesContext context, DataScroller ds, int chunkSize
         wb.finish();
     }
+    
+    private static int getTotalSize(DataScroller ds) {
```
private static -> protected

tandraschko commented on primefaces/src/main/java/org/primefaces/component/datascroller/DataScrollerRenderer.java:
```diff
@@ -192,6 +192,16 @@ protected void encodeScript(FacesContext context, DataScroller ds, int chunkSize
     }
+    
+    private static int getTotalSize(DataScroller ds) {
+        if ( ds.isLazy() ) {
```
our styleguide is `if (ds.isLazy()) {` (remove spaces)

Samuel-StO committed:
Review

Make some change to be compliant with Style Guide
Change accessibility of a method

Samuel-StO committed:
Compliant with style rules

melloware committed:
Update DataScrollerRenderer.java

melloware reviewed (approved):
build is passing now.  LGTM
